google-cloud-testing/spanner-benchwrapper/src/main/proto/spanner.proto

```diff
@@ -18,48 +18,57 @@ package spanner_bench;
 option java_multiple_files = true;
 option java_package = "com.google.cloud.benchwrapper";
 
-message ReadQuery{
-  // The query to use in the read call.
-  string Query = 1;
+message Singer {
+  int64 id = 1;
+  string first_name = 2;
+  string last_name = 3;
+  string singer_info = 4;
 }
 
-message User{
-  string name = 1;
-  int64 age = 2;
+message Album {
+  int64 id = 1;
+  int64 singer_id = 2;
+  string album_title = 3;
 }
 
-message InsertQuery{
-   // The query to use in the insert call.
-  repeated User users = 1;
+message ReadQuery {
+  // The query to use in the read call.
+  string query = 1;
 }
 
-message UpdateQuery{
-  // The queries to use in the update call.
-  repeated string Queries = 1;
+message InsertQuery {
+  // The query to use in the insert call.
+  repeated Singer singers = 1;
+  repeated Album albums = 2;
 }
 
-message EmptyResponse{
+message UpdateQuery {
+  // The queries to use in the update call.
+  repeated string queries = 1;
 }
 
-service SpannerBenchWrapper{
+// TODO(deklerk): Replace with Google's canonical Empty.
```

Comment:
nit: why is this TODO added here? Is there any reason that we cannot use the canonical Empty here that is mentioned?

Reply:
Oops sorry I copied this from an internal proto file as is. I've deleted this comment for now. Since this proto needs to sync with the internal proto, which is also used for the other client libraries, I think I'll leave the `EmptyResponse` here as it is for now so that I don't break everything. I can make another fix in the future for changing to Empty.

Reply:
That sounds reasonable.